dhcpv6/types.go

```diff
@@ -154,79 +154,214 @@ const (
+	OptionNISPServers:                           "OPTION_NISP_SERVERS",
+	OptionNISDomainName:                         "OPTION_NIS_DOMAIN_NAME",
+	OptionNISPDomainName:                        "OPTION_NISP_DOMAIN_NAME",
+	OptionSNTPServerList:                        "SNTP Server List",
+	OptionInformationRefreshTime:                "Information Refresh Time",
+	OptionBCMCSControllerDomainNameList:         "BCMCS Controller Domain Name List",
+	OptionBCMCSControllerIPv6AddressList:        "BCMCS Controller IPv6 Address List",
+	OptionGeoConfCivic:                          "OPTION_GEOCONF",
+	OptionRemoteID:                              "OPTION_REMOTE_ID",
+	OptionRelayAgentSubscriberID:                "Relay-Agent Subscriber ID",
+	OptionFQDN:                                  "FQDN",
+	OptionPANAAuthenticationAgent:               "PANA Authentication Agent",
+	OptionNewPOSIXTimezone:                      "OPTION_NEW_POSIX_TIME_ZONE",
+	OptionNewTZDBTimezone:                       "OPTION_NEW_TZDB_TIMEZONE",
+	OptionEchoRequest:                           "Echo Request",
+	OptionLQQuery:                               "OPTION_LQ_QUERY",
+	OptionClientData:                            "OPTION_CLIENT_DATA",
+	OptionCLTTime:                               "OPTION_CLT_TIME",
+	OptionLQRelayData:                           "OPTION_LQ_RELAY_DATA",
+	OptionLQClientLink:                          "OPTION_LQ_CLIENT_LINK",
+	OptionMIPv6HomeNetworkIDFQDN:                "MIPv6 Home Network ID FQDN",
+	OptionMIPv6VisitedHomeNetworkInformation:    "MIPv6 Visited Home Network Information",
+	OptionLoSTServer:                            "LoST Server",
+	OptionCAPWAPAccessControllerAddresses:       "CAPWAP Access Controller Addresses",
+	OptionRelayID:                               "RELAY_ID",
+	OptionIPv6AddressMOS:                        "OPTION-IPv6_Address-MoS",
+	OptionIPv6FQDNMOS:                           "OPTION-IPv6-FQDN-MoS",
+	OptionNTPServer:                             "OPTION_NTP_SERVER",
+	OptionV6AccessDomain:                        "OPTION_V6_ACCESS_DOMAIN",
+	OptionSIPUACSList:                           "OPTION_SIP_UA_CS_LIST",
+	OptionBootfileURL:                           "OPT_BOOTFILE_URL",
+	OptionBootfileParam:                         "OPT_BOOTFILE_PARAM",
+	OptionClientArchType:                        "OPTION_CLIENT_ARCH_TYPE",
+	OptionNII:                                   "OPTION_NII",
+	OptionGeolocation:                           "OPTION_GEOLOCATION",
+	OptionAFTRName:                              "OPTION_AFTR_NAME",
+	OptionERPLocalDomainName:                    "OPTION_ERP_LOCAL_DOMAIN_NAME",
+	OptionRSOO:                                  "OPTION_RSOO",
+	OptionPDExclude:                             "OPTION_PD_EXCLUDE",
+	OptionVirtualSubnetSelection:                "Virtual Subnet Selection",
+	OptionMIPv6IdentifiedHomeNetworkInformation: "MIPv6 Identified Home Network Information",
```

Comment:
No change in values here, just gofmt moving everything around to be *less* aligned for some reason